agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/AiSemanticAttributes.java

```diff
@@ -106,5 +106,20 @@ public final class AiSemanticAttributes {
   public static final AttributeKey<Boolean> IS_PRE_AGGREGATED =
       AttributeKey.booleanKey("applicationinsights.internal.is_pre_aggregated");
 
+  // Azure Function
+
+  public static final AttributeKey<String> AZ_FN_INVOCATION_ID =
```

Comment:
comment here to explain why these attributes are needed, e.g. to preserve existing behavior